packages/@react-aria/overlays/src/calculatePosition.ts

```diff
@@ -191,18 +191,29 @@ function computePosition(
   let {placement, crossPlacement, axis, crossAxis, size, crossSize} = placementInfo;
   let position: Position = {};
 
-  position[crossAxis] = childOffset[crossAxis] + crossOffset;
+  // button position
+  position[crossAxis] = childOffset[crossAxis];
   if (crossPlacement === 'center') {
+    //  + (button size / 2) - (overlay size / 2)
+    // at this point the overlay center should match the button center
     position[crossAxis] += (childOffset[crossSize] - overlaySize[crossSize]) / 2;
   } else if (crossPlacement !== crossAxis) {
+    //  + (button size) - (overlay size)
+    // at this point the overlay bottom should match the button bottom
     position[crossAxis] += (childOffset[crossSize] - overlaySize[crossSize]);
-  }
+  }/* else {
+    the overlay top should match the button top
+  } */
```

Comment:
I assume this is just explaining what happens when the conditionals don't match.